src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/SetAzureCognitiveServicesAccount.cs

```diff
@@ -74,6 +85,51 @@ public class SetAzureCognitiveServicesAccountCommand : CognitiveServicesAccountB
         [AllowEmptyCollection]
         public string CustomSubdomainName { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Generate and assign a new Cognitive Services Account Identity for this storage account for use with key management services like Azure KeyVault.")]
+        public SwitchParameter AssignIdentity { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        [ValidateNotNullOrEmpty]
+        public IdentityType? IdentityType { get; set; }
```

Comment:
It should just be of type `IdentityType`

Reply:
`ValueFromPipelineByPropertyName = false` is the default, you don't need to specify that

Reply:
I removed `ValueFromPipelineByPropertyName = false`
IdentityType is an enum, and `IdentityType.None` doesn't equal to IdentityType is null.
(`IdentityType.None` means remove Identity, while IdentityType is null means leave it not changed.) I removed ValidateNotNullOrEmpty in case it cases confusing.

Reply:
As the parameter is optional, if user doesn't specify it then its value is `null`. That's why we don't need null-able type parameters.
Please add helpmessage too